deployment/centos/deploy.sh

```diff
@@ -1,14 +1,21 @@
-#!/bin/sh
+#!/bin/bash -e
+
+# change this to the current semver tag, e.g. v1.8.0
+GIT_TAG=master
```

Comment:
Should we turn this into a shell argument?

Reply:
Yes, I thought of that too, so thanks for calling it out 👍  will do.